app/templates/main/browse.html

```diff
@@ -0,0 +1,270 @@
+    <div class="browse__sort-container">
+      <label class="govuk-label" for="sort">
+        Sort by
+      </label>
+      <select class="govuk-select govuk-select__sort-container-select" id="sort" name="sort">
+        <option value="body-a">Transferring body (A to Z)</option>
+        <option value="body-b" selected>Transferring body (Z to A)</option>
+        <option value="series-a">Series (A to Z)</option>
+        <option value="series-b">Series (Z to A)</option>
+        <option value="date-first">Date record transferred (most recent first)</option>
+        <option value="date-last">Date record transferred (oldest first)</option>
+      </select>
+      <button class="govuk-button govuk-button__sort-container-update-button" id="sort-update-button" data-module="govuk-button">
+        Apply
+      </button>
+    </div>
+  </div>
+
+  <div class="govuk-width-container">
+
+    <!-- FILTERS -->
+    <div class="govuk-grid-column-one-third filters-form">
+
+      <div class="browse-filter-container">
+        <div class="browse-filter__header">
+          <p class="govuk-body-l browse__body filters-form__title">Filter within browse</p>
+          <img src="{{ url_for('static', filename='image/filter-icon.svg') }}" class="browse-filter__icon"
+            alt="filter-icon">
+        </div>
+        <p class="govuk-body browse__body">Transferring body</p>
+        <div class="govuk-form-group filters-form__group filters-form__transferring-body-group">
+          <select class="govuk-select govuk-select__filters-form-transferring-body-select" id="sort" name="sort">
+            <option value="published" selected>Choose one...</option>
+            <option value="updated">Arts Council England</option>
+            <option value="views">Food Standards Agency</option>
+            <option value="comments">Foreign Office</option>
+          </select>
+        </div>
+      </div>
+
+      <div class="filters-form__series__container">
+        <p class="govuk-body browse__body">Series</p>
+        <div class="govuk-form-group filters-form__group filters-form__series-group">
+          <input class="govuk-input filters-form__series--input" id="width-5" name="width10" type="text">
+        </div>
+      </div>
+
+      <div class="filters-form__consignment-ref__container">
+        <p class="govuk-body browse__body">Consignment ref</p>
+        <div class="govuk-form-group filters-form__group filters-form__consignment-ref-group">
+          <input class="govuk-input filters-form__consignment-ref--input" id="width-5" name="width10" type="text">
+        </div>
+      </div>
+
+      <div class="filters-form__date__container">
+        <div class="govuk-form-group date-form filters-form__group filters-form__date-from-group">
+          <p class="govuk-body date-form browse__body filters-form__body-date">Date from</p>
+          <fieldset class="govuk-fieldset" role="group" aria-describedby="date-from">
```

Comment:
Added an id for the date from field, we're currently using the DD MM YYYY as hints, I'll check with @Terry-Price to see if hints or labels are preferred. GDS also has some error messaging around this which should be implemented.

The filter functionality is to be implemented so this will be a placeholder for now.